src/components/navbar/navbar.component.tsx

```diff
@@ -94,6 +106,39 @@ const Navbar: React.FC<NavbarProps> = ({
                 >
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
+                  aria-label="Add"
+                  aria-labelledby="Add Button"
+                  name="AddNavButton"
+                  style={{ display: switcherToggler ? "none" : "block" }}
+                  onClick={handleSwitcherClick}
+                >
+                  <AppSwitcher20 />
+                </HeaderGlobalAction>
+                <HeaderGlobalAction
+                  aria-label="Add"
+                  aria-labelledby="Add Button"
+                  name="AddNavButton"
+                  style={{
+                    display: switcherToggler ? "block" : "none",
+                    backgroundColor: "#004144"
+                  }}
+                  onClick={handleCloseClick}
+                >
+                  <Close20 />
+                </HeaderGlobalAction>
+                <HeaderPanel
+                  aria-label="Header Panel"
+                  style={{
+                    display: switcherToggler ? "block" : "none",
+                    height: "12rem",
+                    overFlow: "Hidden"
+                  }}
+                  expanded
+                >
+                  <ExtensionSlot extensionSlotName="global-nav-options"></ExtensionSlot>
```

Comment:
1. The convention is to end with `-slot`
2. Not sure about the name. The left side is already called `nav-menu-slot`, so I am not sure what this is. What is this menu called?

Reply:
Sir, the slot has not yet been created for this.